postman/README.md

```diff
@@ -0,0 +1,37 @@
+# Postman code examples
+
+This directory contains examples for use with Postman
+
+Script | Link |
+--- | --- |
+Create and Schedule Broadcast|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/5a18ec66a11819ae4a3f)
+Find Subscribers Across Lists|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/f6ca23098db76a87c5ad)
+Get Broadcast Details|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/94304813187cfa46ab97)
+Get Broadcast Open Click|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/31f187b357a84c3a2ca8)
+Get Broadcast Stats|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/007cb5e575b093b53444)
+Get Clicks|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/3ef21f19bcf8bc8d7504)
+Get Integrations|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/8f521e5bd78c83b9e8e8)
+Get Tags|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/2e2c3ee4ff07d60146db)
+Get Webforms Detail|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/2e2c3ee4ff07d60146db)
+Manage Custom Fields|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/46ea48d4f90e6595d421)
+Manage Subscriber|
+Move Subscriber|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/37ffdf13b6588329b6da)
+Please follow this tutorial to import this [Postman Collection](https://www.getpostman.com/docs/collections).
+
+## Keys used in APIs that need to be created by the user:
+
+### For all requests
+
+- consumer_key
+- consumer_secret
+- access_token
+- access_secret
+
```

Comment:
To create and schedule a broadcast, all of these environment variables are needed:
* body_html
* body_text
* click_tracking_enabled
* exclude_lists
* facebook_integration
* include_lists
* is_archived
* notify_on_send
* subject
* twitter_integration

If you don't specify them, postman sends the template placeholder as the value:
```
body_html=%7B%7Bbody_html%7D%7D&body_text=test&click_tracking_enabled=%7B%7Bclick_tracking_enabled%7D%7D&exclude_lists=%7B%7Bexclude_lists%7D%7D&facebook_integration=%7B%7Bfacebook_integration%7D%7D&include_lists=%7B%7Binclude_lists%7D%7D&is_archived=%7B%7Bis_archived%7D%7D&notify_on_send=%7B%7Bnotify_on_send%7D%7D&subject=A%20broadcast&twitter_integration=%7B%7Btwitter_integration%7D%7D
```

I recommend hard coding values for all of these since I think it's unreasonable to require this many inputs to run the collection.